ykman/device.py

```diff
@@ -113,6 +114,8 @@ def _parse_capabilities(self, data):
                 self._serial = int(b2a_hex(tlv.value), 16)
             elif YK4_ENABLED_TAG == tlv.tag:
                 self.enabled = int(b2a_hex(tlv.value), 16)
+            elif YK4_FORMFACTOR_TAG == tlv.tag:
+                self._form_factor = FORM_FACTOR(int(b2a_hex(tlv.value), 16))
```

Comment:
The exception where the byte value is not in the `FORM_FACTOR` enum should probably be handled, so that future form factors don't break things. In such a case, `self._form_factor` should probably just be set to the int value.